examples/managed_instance_group/main.tf

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+  region      = "${var.region}"
+}
+
+module "gce-container" {
+  source = "../../../"
+
+  container = {
+    image = "${var.image}"
+  }
+}
+
+resource "tls_private_key" "gce-keypair" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "local_file" "gce-keypair-pk" {
+  content  = "${tls_private_key.gce-keypair.private_key_pem}"
+  filename = "${path.module}/ssh/key"
+}
+
+module "mig" {
```

Comment:
This module also breaks because it depends on the default compute account. Commenting for posterity so we track places where this issue pops up (per https://github.com/terraform-google-modules/terraform-google-project-factory/pull/41), no need for changes.

Reply:
I think that this is a blocker, actually - I'm getting this failure from the managed-instance-group module, even after providing a service account:

```
       Error: Error applying plan:

       1 error(s) occurred:

       * module.mig.google_compute_instance_group_manager.default: 1 error(s) occurred:

       * google_compute_instance_group_manager.default: The resource 'One of [991357374597-compute@developer.gserviceaccount.com]' of type 'serviceAccount' was not found.
```

The documentation at https://www.terraform.io/docs/providers/google/r/compute_instance_group_manager.html doesn't show any parameter related to service accounts, so I don't know how to make this work with the project factory.

Commenting for posterity.